### ISSUE TITLE
So ... where is the code?

### ISSUE THREAD

**Issue by voddan** [Luxoft/evote#1]
The voting system is supposed to be open source, right? So where is all the source code? You know, so that it would be open and everybody could validate it?

**Comment by alexeykoren** [comment]
No, evil Russians wrote it so when you download this code - democracy explodes on your machine
